src/components/infonav.vue

```diff
@@ -0,0 +1,84 @@
+<!--
+Copyright 2025 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <div class="infonav-button dropdown">
+    <router-link v-if="link != null" :to="link">
+      <span :class="icon"></span>
+      {{ title }}
+    </router-link>
+    <button v-else :id="toggleId" type="button" class="btn-link dropdown-toggle" data-toggle="dropdown"
+      aria-haspopup="menu" aria-expanded="false">
+      <span :class="icon"></span>
+      {{ title }}
+      <span class="icon-angle-down"></span>
+    </button>
+    <ul class="dropdown-menu" :aria-labelledby="toggleId">
+      <template v-for="item in items" :key="item.id">
```

Comment:
This could be a slot and the parent could construct the list elements. That could be nice especially for fancy lists with dividers and stuff.